runelite-client/src/main/java/net/runelite/client/config/ChatColorConfig.java

```diff
@@ -490,4 +498,12 @@ default Color transparentExamineHighlight()
 		description = "Color of Usernames in Clan Chat (transparent)"
 	)
 	Color transparentClanUsernames();
+
+	@ConfigItem(
+			position = 87,
```

Comment:
Indent with one tab.

Reply:
These things sometimes slip by I'll fix this in a bit

Reply:
Fixed